subsys/net/ip/Kconfig

```diff
@@ -279,6 +279,22 @@ config NET_TC_NUM_PRIORITIES
 	default NUM_COOP_PRIORITIES if NET_TC_THREAD_COOPERATIVE
```

Comment:
Could you also elaborate in the commit message why we need this feature i.e., why the existing functionality is not enough?

Reply:
updated